File: templates/rspkgdown/dev.R
library(glue)

files <- c("head", "layout") #, "navbar"

for (f in files) {
  reticulate::py_run_string(glue("
  from template_injector import build
  build(
    'templates/pkgdown/rspkgdown/assets/{f}_template.html',
    ['site_assets/components.html',
    'templates/pkgdown/rspkgdown/assets/rspkgdown_components.html'],
    'templates/pkgdown/rspkgdown/inst/pkgdown/BS5/templates/{f}.html'
  )
  "))
}

devtools::install("templates/pkgdown/rspkgdown")

File: themes/rsquartobook/dev.py

# Setup
from template_injector import build


# Build template
build(
    'themes/rsquartobook/rsqb_template.html',
    ['themes/site_components.html', 'themes/rsquartobook/rsqb_components.html'],
    'themes/rsquartobook/assets/rsqb_html.template'
)
